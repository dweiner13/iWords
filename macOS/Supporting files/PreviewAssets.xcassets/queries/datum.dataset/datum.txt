dat.um               VPAR   1 1 NOM S N PERF PASSIVE PPL
dat.um               VPAR   1 1 VOC S N PERF PASSIVE PPL
dat.um               VPAR   1 1 ACC S M PERF PASSIVE PPL
dat.um               VPAR   1 1 ACC S N PERF PASSIVE PPL
do, dare, dedi, datus  V (1st) TRANS  
dat.um               SUPINE 1 1 ACC S N                 
give; dedicate; sell; pay; grant/bestow/impart/offer/lend; devote; allow; make;
dat.um               VPAR   1 1 NOM S N PERF PASSIVE PPL
dat.um               VPAR   1 1 VOC S N PERF PASSIVE PPL
dat.um               VPAR   1 1 ACC S M PERF PASSIVE PPL
dat.um               VPAR   1 1 ACC S N PERF PASSIVE PPL
dat.um               SUPINE 1 1 ACC S N                 
surrender/give over; send to die; ascribe/attribute; give birth/produce; utter;
dat.um               N      2 2 NOM S N                 
dat.um               N      2 2 VOC S N                 
dat.um               N      2 2 ACC S N                 
datum, dati  N (2nd) N  
present/gift; that which is given; debit; [~ dandis => w/all supplied];
dat.um               N      4 1 ACC S M                 
datus, datus  N (4th) M    Early  veryrare
act of giving;
*
