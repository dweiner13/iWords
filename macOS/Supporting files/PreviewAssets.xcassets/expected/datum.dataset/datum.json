[{"definition":{"_0":{"words":[{"meaning":"give; dedicate; sell; pay; grant\/bestow\/impart\/offer\/lend; devote; allow; make;","expansion":{"verb":{"_0":"do, dare, dedi, datus","_2":["trans."],"_1":1}}}],"truncated":false,"possibilities":[{"verbParticiple":{"_0":{"gender":"N","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"NOM"}}},{"verbParticiple":{"_0":{"gender":"N","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"VOC"}}},{"verbParticiple":{"_0":{"gender":"M","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"ACC"}}},{"verbParticiple":{"_0":{"gender":"N","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"ACC"}}},{"supine":{"_0":{"gender":"N","conjugation":1,"case":"ACC","number":"S","variety":1,"text":"dat.um"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"surrender\/give over; send to die; ascribe\/attribute; give birth\/produce; utter;"}],"truncated":false,"possibilities":[{"verbParticiple":{"_0":{"gender":"N","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"NOM"}}},{"verbParticiple":{"_0":{"gender":"N","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"VOC"}}},{"verbParticiple":{"_0":{"gender":"M","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"ACC"}}},{"verbParticiple":{"_0":{"gender":"N","number":"S","variety":1,"conjugation":1,"tense":"PERF","text":"dat.um","voice":"PASSIVE","case":"ACC"}}},{"supine":{"_0":{"gender":"N","conjugation":1,"case":"ACC","number":"S","variety":1,"text":"dat.um"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"present\/gift; that which is given; debit; [~ dandis => w\/all supplied];","expansion":{"noun":{"_1":2,"_0":"datum, dati","_3":[],"_2":"N"}}}],"truncated":false,"possibilities":[{"noun":{"_0":{"declension":2,"case":"NOM","number":"S","gender":"N","variety":2,"text":"dat.um"}}},{"noun":{"_0":{"declension":2,"case":"VOC","number":"S","gender":"N","variety":2,"text":"dat.um"}}},{"noun":{"_0":{"declension":2,"case":"ACC","number":"S","gender":"N","variety":2,"text":"dat.um"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"act of giving;","expansion":{"noun":{"_1":4,"_0":"datus, datus","_3":["Early","very rare"],"_2":"M"}}}],"truncated":false,"possibilities":[{"noun":{"_0":{"declension":4,"case":"ACC","number":"S","gender":"M","variety":1,"text":"dat.um"}}}]}}}]